flight/Modules/Stabilization/stabilization.c

```diff
@@ -324,7 +324,35 @@ static void stabilizationTask(void* parameters)
 
 					break;
 
-				case STABILIZATIONDESIRED_STABILIZATIONMODE_ATTITUDE:
+			case STABILIZATIONDESIRED_STABILIZATIONMODE_ACRO:
+					// this implementation is based on the Openpilot/Librepilot Acro+ flightmode
+					// and our existing rate & MWRate flightmodes
+					if(reinit)
+							pids[PID_RATE_ROLL + i].iAccumulator = 0;
+
+					// The unscaled input (-1,1)
+					float *raw_input = &stabDesired.Roll;
```

Comment:
Notice here https://github.com/librepilot/LibrePilot/blob/next/flight/modules/Stabilization/innerloop.c#L298 how LibrePilot divide out the constant to recover the raw stick position. This feels a bit hacky to me though. I prefer how it is handled in MWRate mode. Just do not apply the scaling in ManualControl so the input really _is_ from -1,1. Then below on https://github.com/TauLabs/TauLabs/pull/1816/files#diff-3c1ff3789e117137da52548be01cc9f4R340 you can apply the scaling to compute ```rateDesiredAxis[i]```